dcrlibwallet.go

```diff
@@ -2,70 +2,58 @@ package dcrlibwallet
 
 import (
 	"context"
-	"fmt"
-	"os"
 	"path/filepath"
 
-	"github.com/asdine/storm"
-	"github.com/decred/dcrwallet/errors"
-	"github.com/decred/dcrwallet/netparams"
-	"github.com/decred/dcrwallet/wallet"
-	"github.com/decred/dcrwallet/wallet/txrules"
+	"github.com/decred/dcrwallet/errors/v2"
+	"github.com/decred/dcrwallet/wallet/v3"
+	"github.com/decred/dcrwallet/wallet/v3/txrules"
+	"github.com/raedahgroup/dcrlibwallet/internal/loader"
+	"github.com/raedahgroup/dcrlibwallet/internal/netparams"
 	"github.com/raedahgroup/dcrlibwallet/txindex"
 	"github.com/raedahgroup/dcrlibwallet/utils"
-	"go.etcd.io/bbolt"
 )
 
-const logFileName = "dcrlibwallet.log"
+type Properties struct {
```

Comment:
`Properties` is definitely not a good type name. If the properties of a struct are properties of a person, then typically the struct name should be `Person`. What entity has these properties that are defined in this struct? The name of that entity should be the name of this struct.

I strongly believe `Wallet` is a fine name. Then you can do stuff like `wallet.ID`, `wallet.Name`, `wallet.Seed` and it just all makes sense.

Using `Wallet` as a type name would not conflict with any other `Wallet` type defined in any other package, because there'd obviously be a clear difference between `dcrwallet.Wallet` for example and `dcrlibwallet.Wallet` just as there is between the `Error` type in go's inbuilt package and `errors.Error` from `github.com/decred/dcrwallet/errors`.